src/kw_include.sh

```diff
@@ -37,14 +39,21 @@ function include()
   fi
 
   varname="$(get_varname "$fullpath")"
+  hashname="${varname%"_IMPORTED"}_HASH"
```

Comment:
Won't this give us problems if we have a third+ file with the same name?

Reply:
you're quite right on that! I've actually spent a little time thinking about that since I committed this, and now I think this might be sorted :)